src/thin_client.rs

```diff
@@ -470,6 +470,27 @@ pub fn poll_gossip_for_leader(leader_ncp: SocketAddr, timeout: Option<u64>) -> R
     Ok(leader.unwrap().clone())
 }
 
+pub fn retry_get_balance(
+    client: &mut ThinClient,
+    bob_pubkey: &Pubkey,
+    expected: Option<i64>,
+) -> Option<i64> {
+    const LAST: usize = 30;
+    for run in 0..=LAST {
+        let out = client.poll_get_balance(bob_pubkey);
+        if expected.is_none() || run == LAST {
+            return out.ok();
+        }
+        trace!("retry_get_balance[{}] {:?} {:?}", run, out, expected);
+        if let (Some(e), Ok(o)) = (expected, out) {
+            if o == e {
```

Comment:
Please pick better variable names than `o` and `e`.  This if statement is too hard to understand: `if o == e {` (eg, what is o and e and why would they be equal?)